### PR TITLE
Update Swift pipeline w byo cluster

### PR DESCRIPTION
**Pull request by camrynl**
**Reason for Change**:

Current e2e tests are utilizing an existing cluster. Each PR waits for the cluster to be available before running tests.
Adding byo cluster feature to pipeline so each PR can run tests in parallel using its own cluster.

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

**Comment by camrynl** [comment]
PR still in progress. Can't convert to draft since I'm testing the pipeline. I'll notify when to review, ty :)

File: .pipelines/singletenancy/aks-swift/e2e-step-template.yaml [hunk 1]
```diff
@@ -26,18 +26,17 @@ steps:
 
   - task: AzureCLI@1
     inputs:
-      azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+      azureSubscription: "Azure Container Networking - Test"
```
Comment (rbtr) [inline .pipelines/singletenancy/aks-swift/e2e-step-template.yaml:29]: there is a pipeline var for this Sub, we should use it

File: cns/azure-cns.yaml [hunk 2]
```diff
@@ -96,7 +96,7 @@ spec:
           effect: NoSchedule
       containers:
         - name: cns-container
-          image: mcr.microsoft.com/containernetworking/azure-cns:v1.4.7
+          image: mcr.microsoft.com/containernetworking/azure-cns:v1.4.29
```
Comment (rbtr) [inline cns/azure-cns.yaml:99]: Can you revert this? I think this manifest is used by partners, and is unrelated to this change anyway

**Review by rbtr — CHANGES_REQUESTED** [review]

**Comment by rbtr** [comment]
You can also remove this line https://github.com/Azure/azure-container-networking/blob/637d5affe966f2aadf0240e64afc0cc34e481696/.pipelines/singletenancy/aks-swift/e2e-job-template.yaml#L9 which currently prevents this test job from running in parallel 🙂

**Review by rbtr — APPROVED** [review]
🚀🚀

**Pull request merged**